docs/pages/operations-guide.md

```diff
@@ -320,7 +320,7 @@ name: secret-app/instance1
 pods:
```

Comment:
@sascala Yes, incorporating some of it here makes sense definitely. 

You can reference the secret as a file for increased security from other processes, or if your service needs to read secrets from files mounted in the container.

Referencing a file-based secret can be particularly useful for:

Kerberos keytabs or other credential files.
SSL certificates.
Configuration files with sensitive data.